### PR TITLE
[SSPROD-9138] feat: multiarch builds

### PR DESCRIPTION
**Pull request by cgeers**

### CODE REVIEW

File: Jenkinsfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 pipeline {
     agent {
-        label 'builder-backend-j8 || new-builder-backend-j8'
```
Comment (cgeers) [inline Jenkinsfile:3]: what was special about this builder?

Comment (nkraemer-sysdig) [reply Jenkinsfile:3]: Nothing, it was a general purpose builder. Certain functions needed dedicated builders (like the agent), so we started adding builder labels to all jobs

**Review by noahkraemer — APPROVED** [review]

File: makefile-sysdig [hunk 2]
```diff
@@ -1,16 +1,18 @@
...
-push-dependency-image:
-	$(DOCKER) push $(DEPENDENCY_IMAGE_NAME)
+ifndef PUSH
+	PUSH_FLAG := ""
+else
+	PUSH_FLAG := "--push"
+endif
 
-delete-dependency-image:
-	$(DOCKER) rmi $(DEPENDENCY_IMAGE_NAME)
```
Comment (nkraemer-sysdig) [inline makefile-sysdig:16]: Do we not need the delete target anymore? Do we clean up images on the builder another way?

Comment (cgeers) [reply makefile-sysdig:16]: > Do we not need the delete target anymore? Do we clean up images on the builder another way?

I don't believe anything else is needed for cleanup. buildx just builds, images aren't "loaded" into the docker graph dir unless requested (and we aren't). The removal/creation of the buildx worker container each time is all we need (AFAIK).

Note that I considered a multibranch approach where builders would be namespaced and could run independently, but decided wasn't critical.

**Pull request merged**